plugins/hls-code-range-plugin/test/Ide/Plugin/CodeRangeTest.hs

```diff
@@ -50,5 +50,66 @@ testTree =
                         ( SelectionRange (Range (Position 1 1) (Position 5 10)) Nothing
                         )
                     )
-            ]
+            ],
+
+        testGroup "findFoldingRanges" $
```

Comment:
This could also be in its own module: this module is `CodeRangeTest`, perhaps create `FoldingRangeTest`. Not super important though.

Reply:
Module CodeRangeTest only had tests about selection range. 

So I thought of it as testing CodeRange implementation on selection and folding ranges.

Should I keep it the same or change it?

Reply:
Oh I see, good point. I forgot that "code range" is more general than "selection range"!